client/src/components/LogoHeader/LogoHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import Box from '@material-ui/core/Box';
+import useStyles from './useStyles';
+import { Typography } from '@material-ui/core';
+
+interface Props {
+  logo: string;
+}
+
+const LogoHeader = ({ logo }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.logo}>
+      {/*TODO: Add link*/}
+      <img width="50px" src={logo} />
+      <Typography className={classes.logoText} component="h1" variant="h4">
+        LovingSitter.
+      </Typography>
+    </Box>
+  );
```

Comment:
This is a nice component! Although, we should have a `png` logo among the mocks in the Google Drive folder, we could probably use that instead of using `Typography` here.